### ISSUE TITLE
Missing validation when checking domain settings

### ISSUE THREAD

**Issue by digininja** [Raikia/FiercePhish#66]
On the "Check Email Settings" page ( /emails/check ), if you put in a value that isn't a domain name when checking the domain, the site redirects to the dashboard rather than staying on the page and giving an error.

**Comment by Raikia** [comment]
Yeah, that's due to silly error handling here: https://github.com/Raikia/FiercePhish/blob/master/resources/views/emails/check_settings.blade.php#L177

This won't exist in v2.0 so it will disappear then.  Thanks for the report!
